profiles/base/package.use.stable.mask

```diff
@@ -4,6 +4,10 @@
 # This file requires eapi 5 or later. New entries go on top.
 # Please use the same syntax as in package.use.mask
 
+# Bernd Waibel <waebbl@gmail.com> (13 May 2019)
+# needs build and runtime testing
```

Comment:
That's not a particularly good mask message even if you'll find it used in many places (but it is more or less implied anyway by being in this file; or `~arch`, for that matter). I couldn't help but notice the ebuild telling that also `prman` is still experimental. Should it also be stable-masked then? The message should then contain the actual reason for the masking.

Reply:
Thanks for the hint. I know it's not a good message, but I just looked at the common way in the file. Not a particularly good habit on my side either... I changed the message to hopefully be more meaningful.
The `prman` USE flag is told experimental in the ebuild, but it could theoretically be removed, and I'm probably doing this at the next bump. I have a renderman ebuild in my overlay and can test the addon with this. It has a basic working at least, but I don't had the time to thoroughly test it, so the info on being experimental is still there.